pkg/operator/controllers/guardrails/policies/gktemplates-src/library/common.rego

```diff
@@ -113,31 +108,35 @@ privileged_ns = {
   "openshift-host-network",
   "openshift-image-registry",
   "openshift-ingress",
-  "openshift-ingress-canary",
   "openshift-ingress-operator",
-  "openshift-insights",
-  "openshift-kni-infra",
   "openshift-kube-apiserver",
   "openshift-kube-apiserver-operator",
   "openshift-kube-controller-manager",
   "openshift-kube-controller-manager-operator",
   "openshift-kube-scheduler",
   "openshift-kube-scheduler-operator",
-  "openshift-kube-storage-version-migrator",
-  "openshift-kube-storage-version-migrator-operator",
   "openshift-machine-api",
   "openshift-machine-config-operator",
-  "openshift-marketplace",
   "openshift-monitoring",
   "openshift-multus",
-  "openshift-network-diagnostics",
   "openshift-network-operator",
   "openshift-oauth-apiserver",
-  "openshift-openstack-infra",
   "openshift-operators",
   "openshift-operator-lifecycle-manager",
-  "openshift-ovirt-infra",
-  "openshift-sdn",
   "openshift-service-ca",
-  "openshift-service-ca-operator"
-}
\ No newline at end of file
+  "openshift-service-ca-operator",
+  # "openshift-kube-storage-version-migrator",
+  # "openshift-kube-storage-version-migrator-operator",
+  # "openshift-network-diagnostics",
+  # "openshift-openstack-infra",
+  # "openshift-marketplace",
+  # "openshift-ingress-canary",
+  # "openshift-insights",
+  # "openshift-kni-infra",
+  # "openshift-cluster-csi-drivers",
+  # "openshift-cluster-node-tuning-operator",
+  # "openshift-cluster-samples-operator",
+  # "openshift-config",
+  # "openshift-ovirt-infra",
+  "openshift-sdn"
```

Comment:
It would be more readable to make it in alphabetical order?

Reply:
these removed items were temporarily put here, once confirmed they will be removed from code base as this file will be included in many other policies.